migration-lock-timeout.gemspec

```diff
@@ -29,10 +29,11 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "appraisal"
   spec.add_development_dependency "database_cleaner"
-  spec.add_development_dependency "pg", "< 1.0"
+  spec.add_development_dependency "pg", "~> 1.1"
   spec.add_development_dependency "pry"
   spec.add_development_dependency "rake", "~> 11.2.2"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "byebug"
```

Comment:
@cwjenkins would you mind dropping the byebug dependency? That feels more like a developer preference than a needed item. We are looking at dropping pry as well.

Reply:
Sure, figured having an actual debugger helps during development. Believe they'll have one ('debug') in std lib in 3.1.